docs/Config_Reference.md

```diff
@@ -1835,6 +1835,42 @@ z_offset:
 #   See the "probe" section for more information on the parameters above.
 ```
 
+### [load_cell_probe]
+
+Load-cell based probe. One may define this section (instead of a probe
+section) to enable a load-cell based probe. See
+[Load-cell probe guide](LoadCellProbe.md) and
+[command reference](G-Codes.md#load_cell_probe) for further
+information. Some of the parameters are determined in a semi-automatic
+[calibration procedure](LoadCellProbe.md#setup-for-new-printer-models).
+
+```
+[load_cell_probe]
+#
+# General parameters:
+#
+adc:
+#   ADC pin which digitizes the load cell signal.
+adc_rate:
+#   Rate (in samples per second) at which to request the ADC samples.
+max_abs_force:
+#   Maximum absolute value of measured force (in chosen unit), if exceeded
```

Comment:
FWIW, I find it confusing for the reference to be vague on units.  I think it would be more clear to just say `unit of grams`.